src/app/redux/api/auth.ts

```diff
@@ -0,0 +1,86 @@
+import { invoke, APP_HASH, APP_ID, makePasswordHash } from 'helpers/Telegram';
```

Comment:
api folder is going to be huge, really bigger than modules folder itself. I don`t know where is should be, but now modules folder seems to be pure store implemenation directory